spec/features/article_user_feature_spec.rb

```diff
@@ -28,6 +29,8 @@
     expect(page).to have_content(article.user.username)
     expect(page).to have_content("First Article Title")
     expect(page).to have_content("This is the first official Article of FantasyMetrix.")
+    expect(page).to_not have_content("Edit Article")
+    expect(page).to_not have_content("Delete Article")
   end
```

Comment:
Consider using `have_link` in place of `have_content` on lines 21, 32, 33. The `have_link` makes it a little clearer as to what's being tested.